docs/source/status.rst

```diff
@@ -243,47 +243,51 @@ support/not support individual features.
+| GetSqlInfo                                 | ✓     | ✓     | ✓     |            | ✓ (1) |       |       |       |
 +--------------------------------------------+-------+-------+-------+------------+-------+-------+-------+-------+
-| GetTables                                  | ✓     | ✓     | ✓     |            |       |       |       |       |
+| GetTables                                  | ✓     | ✓     | ✓     |            | ✓     |       |       |       |
 +--------------------------------------------+-------+-------+-------+------------+-------+-------+-------+-------+
-| GetTableTypes                              | ✓     | ✓     | ✓     |            |       |       |       |       |
+| GetTableTypes                              | ✓     | ✓     | ✓     |            | ✓     |       |       |       |
 +--------------------------------------------+-------+-------+-------+------------+-------+-------+-------+-------+
-| GetXdbcTypeInfo                            | ✓     | ✓     | ✓     |            |       |       |       |       |
+| GetXdbcTypeInfo                            | ✓     | ✓     | ✓     |            | ✓     |       |       |       |
 +--------------------------------------------+-------+-------+-------+------------+-------+-------+-------+-------+
-| PreparedStatementQuery                     | ✓     | ✓     | ✓     |            |       |       |       |       |
+| PreparedStatementQuery                     | ✓     | ✓     | ✓     |            | ✓     |       |       |       |
 +--------------------------------------------+-------+-------+-------+------------+-------+-------+-------+-------+
-| PreparedStatementUpdate                    | ✓     | ✓     | ✓     |            |       |       |       |       |
+| PreparedStatementUpdate                    | ✓     | ✓     | ✓     |            | ✓     |       |       |       |
 +--------------------------------------------+-------+-------+-------+------------+-------+-------+-------+-------+
 | StatementSubstraitPlan                     | ✓     | ✓     |       |            |       |       |       |       |
 +--------------------------------------------+-------+-------+-------+------------+-------+-------+-------+-------+
-| StatementQuery                             | ✓     | ✓     | ✓     |            |       |       |       |       |
+| StatementQuery                             | ✓     | ✓     | ✓     |            | ✓     |       |       |       |
 +--------------------------------------------+-------+-------+-------+------------+-------+-------+-------+-------+
-| StatementUpdate                            | ✓     | ✓     | ✓     |            |       |       |       |       |
+| StatementUpdate                            | ✓     | ✓     | ✓     |            | ✓     |       |       |       |
 +--------------------------------------------+-------+-------+-------+------------+-------+-------+-------+-------+
 
+Notes:
+
+* \(1) GetSqlInfo filtering is not currently supported in C#.
```

Comment:
What does the "filtering" mean?
It seems that `GetSqlInfo` doesn't have filtering feature.

Reply:
Sorry about that I was mixing up our implementation with the base support provided by Arrow Flight SQL.  Our implementation doesn't allow a `CommandGetSqlInfo` `Info` field to be populated when the command is executed.  I've seen implementations in other languages where the `Info` field is used as a metadata filter list.  However, the C# base implementation does not impose any restrictions.

I fixed the documentation.  Thanks for the catch!

Reply:
I see. :-)